gsa/src/web/pages/reports/alertactions.js

```diff
@@ -152,16 +150,15 @@ class AlertActions extends React.Component {
   }
 
   onAlertCreated(response) {
-    const {alerts} = this.state;
-    const {gmp} = this.props;
-    gmp.alert.get({id: response.data.id}).then(resp => {
-      const alert = resp.data;
-      this.setState({
-        alerts: [alert, ...alerts],
-        alertId: alert.id,
+    this.props.loadAlerts().then(() => {
+      this.props.loadAlert(response.data.id).then(resp => {
+        const alert = resp.data;
+        this.setState({
+          alertId: alert.id,
```

Comment:
Why not use the passed alert id here (currently from response object of onAlertCreated)?